### ISSUE TITLE
Lavacharts loading after all page scripts loaded

### ISSUE THREAD

**Issue by davemg3** [kevinkhill/lavacharts#252]
Hi,

It might not be a Lavachart question and more a general development question so sorry if the case.
  
I noticed all my lavacharts being loaded only after all other scripts on page are loaded. For example I have a Google map with markers which takes time to load and this slowdown all my lavacharts  display,coming after.

Lavacharts use Facade for usage in code and not a script that i can choose the hierarchy vs other scripts.
Is there a way to put forward the load of my Lavacharts vs other long script?

**Comment by kevinkhill** [comment]
Yes! absolutely you can. Sorry for the delayed reply, I am very busy outside of this hobby project.

You can use the [library option](https://github.com/kevinkhill/lavacharts/blob/1e4d056519a933e2406699d956b2bd8a27150955/src/Laravel/config/lavacharts.php#L23) `'auto_run' => false` (This is accessed either through the published config in Laravel or passed to the constructor when instansiating Lavacharts if you are not using Laravel)

This will [prevent Lavacharts from running](https://github.com/kevinkhill/lavacharts/blob/1e4d056519a933e2406699d956b2bd8a27150955/javascript/src/lava.entry.js#L40) in-place, where you put the render call. You can then call `lava.run()` when it is appropriate for your application.

**Comment by kevinkhill** [comment]
I'm going to close this, just because it is so stale. This project is not abandoned, just in a stable state for now. I am always happy to help and please open a new issue if needed.

Thank you for using Lavacharts!
